src/app/components/Grid/README.md

```diff
@@ -0,0 +1,9 @@
+# Grid
+
+## Usage
+
+When using one of the extended Grid components, e.g. `FrontPageGrid`, if you wish to extend it using the `as` prop, you'll need to use `forwardedAs`. This prop enables you to pass down the element down further than one level.
```

Comment:
👍 Good to add this extra note for usage. We should likely add this in the original [`psammead-grid`](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-grid/README.md) documentation.

Reply:
Will include this in the next PR on psammead-grid